pages/index.js

```diff
@@ -167,6 +168,14 @@ function HomePage() {
           </div>
         </section>
       </Container>
+      <Container className="text-center py-6 pb-20" wide>
+        <h3 className="text-primary-800 text-3xl font-bold md:text-4xl mb-4">Supported by</h3>
+        <p className="mt-3 max-w-2xl mx-auto text-l leading-7 text-gray-500 sm:mt-4 pb-4">
+          The following companies support us by letting us use their products for free. Interested in supporting us too?
+          <a href="mailto:info@asyncapi.io" target="_blank" className="text-primary-600" rel="noreferrer"> Email </a> for more info.
```

Comment:
```suggestion
          <a href="mailto:info@asyncapi.io" target="_blank" className="text-primary-600" rel="noreferrer"> Email </a> us for more info.
```

in text proposed by @alequetzalli email address was not a hyperlink and indicated "who to communicate with" and now once it is a hyperlink under `Email` we lost this "who", this is why I propose to add `us`

Reply:
Yeah, got your point :)